neurom/fst/tests/test_get_features.py

```diff
@@ -321,39 +321,39 @@ def test_total_length_pop():
     feat = 'total_length'
 
     nt.ok_(np.allclose(fst.get(feat, POP),
-                       [840.47744821, 418.83424432, 13250.82577394]))
+                       [840.68522362, 418.83424432, 13250.82577394]))
```

Comment:
Yes, that would explain it. So this is related to the conversion to h5. Yes, it is outside the scope of this pull request. However, it is nice to document why this changed and to keep it in mind for future reference. Thanks!